docs/installation.md

```diff
@@ -2,30 +2,107 @@
+to register nodes in a Rancher management cluster and fetch the installation configuration.
+
+## Elemental Operator Installation
+
+The {{elemental.operator.name}} is responsible for managing the Elemental versions and
+maintaining a machine inventory to assist with edge or baremetal installations. {{elemental.operator.name}}
+requires a cluster including the Rancher Manager and it can be installed with a helm chart.
+
+See {{elemental.operator.name}} [helm chart reference](elementaloperatorchart-reference.md) for install,
+uninstall, upgrade and configuration details.
+
+## Prepare Kubernetes Resources
+
+Once the {{elemental.operator.name}} is up and running within the management cluster a couple of kuberentes
+resources are required in order to prepare an Elemental based cluster deployment.
+
+* [MachineInventorySelectorTemplate](machineinventoryselectortemplate-reference.md): 
+  This resource identifies the criteria to match registered boxes (listed as part of the MachineInventory)
+  against available Rancher 2.6 Clusters. As soon as there is a match the matching kubernetes cluster takes
+  ownership of the registered box.
+  
+* [MachineRegistration](machineregistration-reference.md):
+  This resource defines OS deployment details for any machine attempting to register. The machine
+  registration is the entrance for Elemental nodes as it handles the authentication (based on TPM),
+  the Elemental Teal deployment and the node inclusion into to the MachineInventory so it can be added
+  to a cluster when there is a match based on a MachineInventorySelectorTemplate. The MachineRegistration
+  object includes the machine registration URL that nodes use to register against it.
+
+A Rancher Cluster resource is also required to deploy Elemental, it can be manually created as exemplified in
+the [Quick Start](quickstart.md) guide or created from the Rancher 2.6 UI.
+
+
+## Perpare Installation Media
+
+The installation media is the media that will be used to kick start an Elemental Teal deployment. Currently
+the supported media is a live ISO. The live ISO must include the registration configuration yaml hence it must
+crafted once the MachineRegistration is created. The installation media is created by using the `elemental-iso-build`
+helper script (see [quick start](quickstart.md#preparing-the-iso) guide)
+or by using the `elemental build-iso` command line utility included as part of the
+[Elemental Toolkit]({{elemental.toolkit.url}}/docs/creating-derivatives/build_iso).
+
+Within MachineRegistration only a subset of OS installation parameters can be configured, all available parameters listed
+at MachineRegistration reference page.
 
-## Elemental Partition Table
+In order to configure the installation beyond the common options provided within the
+[`elemental.install`](machineregistration-reference.md#configelementalinstall) section a `/etc/elemental/config.yaml`
```

Comment:
We dont have any way of including this yet no?

Reply:
Yes you can if you cook yourself the ISO but I did not want to get into details here as this is likely to change in behalf of #263 

I started documenting a manual process but soon I realized I was getting into rabbit hole.